documentation/examples/15-composition/03-slot-props/Hoverable.svelte

```diff
@@ -10,6 +10,6 @@
 	}
 </script>
 
-<div on:mouseenter={enter} on:mouseleave={leave}>
+<div on:mouseenter={enter} on:mouseleave={leave} role="button" tabindex='0'>
```

Comment:
We should ignore this warning for now, since this example relies on hovering and can't be made keyboard accessible